index_V1

```diff
@@ -57,6 +87,14 @@
 					
 					<div class="text" style="grid-column: 1; background-color: grey; margin: 0vw 1vw;"></div>
 					
+					<div onclick="location.href='query_1.php';" class="circle_1" style="cursor: pointer; grid-column: 2; background-color: #e6e6e6; margin: 1vw 1vw; transition: 0.6s;"></div>
+					
+					<div onclick="location.href='query_1.php';" class="circle_2" style="cursor: pointer; grid-column: 3; background-color: #e6e6e6; margin: 1vw 1vw; transition: 0.6s;"></div>
+					
+					<div onclick="location.href='query_1.php';" class="circle_3" style="cursor: pointer; grid-column: 4; background-color: #e6e6e6; margin: 1vw 1vw; transition: 0.6s;"></div>
+					
+					<div onclick="location.href='query_1.php';" class="circle_4" style="cursor: pointer; grid-column: 5; background-color: #e6e6e6; margin: 1vw 1vw; transition: 0.6s;"></div>
+				
```

Comment:
These are the divs for the bottom half of the four album covers. This leads the user to the "For You" queries.